games.app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { getCategories } = require("./games.controller");
+const app = express();
+
+app.get("/api");
+app.get("/api/categories", getCategories);
+
```

Comment:
You need a path to catch the 404 error in app.js. You need to add a 500 error handler too.